lib/page/needed.rb

```diff
@@ -0,0 +1,31 @@
+module Page
+  class Needed
+    def initialize
+      make_client
+    end
+
+    def to_find
+      @client.issues 'everypolitician/everypolitician-data', labels: 'New Country,To Find'
+    end
+
+    def to_scrape
+      @client.issues 'everypolitician/everypolitician-data', labels: 'New Country,To Scrape'
+    end
+
+    def to_finish
+      @client.issues 'everypolitician/everypolitician-data', labels: 'New Country,3 - WIP'
+    end
+
+    private
+
+    def make_client
```

Comment:
Rather that making this a command that you call from the constructor to set an instance variable, it would be cleaner to just have it be a simple `client` method that creates it on demand and returns it, and make all the calls go through the method, rather than access the variable.